spec/spec_helper.rb

```diff
@@ -2,11 +2,19 @@
 require "coveralls"
 
 SimpleCov.formatter = Coveralls::SimpleCov::Formatter
-SimpleCov.start 
+SimpleCov.start
 
 require File.dirname(__FILE__) + '/../lib/require_all.rb'
 
 module SpecHelper
+  def fixture_path(fixture_name, relative_dir = File.dirname(__FILE__))
```

Comment:
Small refactoring: `File.dirname(__FILE__)` can be more succinctly written as `__dir__`

Reply:
If it passes on CI, then there's no problem. I'm pretty sure it will pass there with this change too.